shared/android/build.gradle

```diff
@@ -77,5 +77,6 @@ allprojects {
 
         google()
         jcenter()
+        maven { url 'https://jitpack.io' }
```

Comment:
What's this for?

Reply:
this is based on the shipping template and we already had this before